src/logic/tests/calculate.test.js

```diff
@@ -0,0 +1,54 @@
+// write unit tests for calculate.js
+
+import calculate from '../calculate';
+
+// test for AC button
+
+test('AC button', () => {
+  const obj = {
+    total: 5,
+    next: 5,
+    operation: '+',
+  };
+  const buttonName = 'AC';
+  const result = calculate(obj, buttonName);
+  expect(result).toEqual({
+    total: null,
+    next: null,
+    operation: null,
+  });
+});
+
+// test for equal button
+
+test('equal button', () => {
+  const obj = {
+    total: 5,
+    next: 5,
+    operation: '+',
```

Comment:
- [ ] All the functionalities & scenarios of the calculate.js module should be covered using Jest.